crates/benchmark-runner/src/stateless_validator.rs

```diff
@@ -61,6 +62,16 @@ pub fn stateless_validator_inputs(
                 output: ProgramOutputVerifier {
                     block_hash: bw.block_and_witness.block.hash_slow().0,
                     parent_hash: bw.block_and_witness.block.parent_hash.0,
+                    versioned_hashes_hash: calculate_versioned_hashes_hash(
+                        &bw.block_and_witness.chain_config,
+                        &bw.block_and_witness.block,
+                    ),
+                    parent_beacon_block_root: bw
+                        .block_and_witness
+                        .block
+                        .parent_beacon_block_root
+                        .map(|root| root.0),
+                    requests_hash: bw.block_and_witness.block.requests_hash.map(|h| h.0),
```

Comment:
As a reminder `ProgramOutputVerifier` contains the expected values for public input checking. Here, we add the values expected for the new inputs.

This would be akin to what a zkCL verifier would have to do to pass the corresponding public inputs.